### PR TITLE
Allow definition of g:slime_cell_delimeter using autocommands

### PR DESCRIPTION
**Pull request by mroavi**
Fixes https://github.com/jpalardy/vim-slime/issues/256

### CODE REVIEW

**Comment by mroavi** [comment]
I initialize `g:slime_cell_delimeter` with an empty string. If `slime#send_cell` is called without the user assigning a delimiter string other than an empty string, a custom error will be echoed. Otherwise, the function will run as before.

File: autoload/slime.vim [hunk 1]
```diff
@@ -349,16 +353,20 @@ function! slime#send_lines(count) abort
...
+    " line before delimiter or bottom of file
+    let line_end = line_end ? line_end - 1 : line("$")
 
-  if line_ini <= line_end
-    call slime#send_range(line_ini, line_end)
+    if line_ini <= line_end
+      call slime#send_range(line_ini, line_end)
+    endif
+  else
+    echoerr "g:slime_cell_delimiter is empty"
```
Comment (jpalardy) [inline autoload/slime.vim:369]: would it make sense to "prompt" the user here?

**Comment by mroavi** [comment]
> if you're willing, I can open a PR against your branch and we can iterate on this
> 
> or, I can start a branch on my side and pull your changes

Sure. Let's go for the second option.

File: autoload/slime.vim (hunk 1 above)
Comment (mroavi) [reply autoload/slime.vim:369]: I would rather not. A trend that I've seen in many vim plugins is that the state of the program should stay in sync with the state defined in the configuration files.

**Comment by jpalardy** [comment]
have a look at #258

**Pull request merged**